test/components/Theme.spec.ts

```
@@ -0,0 +1,238 @@
+    [
+      'with theme not affecting unrelated component',
+      {
+        props: { ui: { badge: { slots: { base: 'rounded-[1.234rem]' } } } },
+        slots: { default: () => h(Button, { label: 'Button' }) }
+      } satisfies CaseOptions,
+      []
+    ]
+  ])('renders %s correctly', async (nameOrHtml: string, options: CaseOptions, contains: string[] = []) => {
+    const html = await ComponentRender(nameOrHtml, options, Theme)
+    expect(html).toMatchSnapshot()
+    contains.forEach(c => expect(html).toContain(c))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)